docs/synapse/userguides/storm_ref_data_mod.rstorm

```diff
@@ -506,34 +640,47 @@ Add Tag Timestamps or Tag Properties
 
 Synapse supports the use of :ref:`tag-timestamps` and :ref:`tag-properties` to provide additional context to tags where appropriate.
 
+.. TIP::
+  
+  You can optionally use the :ref:`edit-try` ( ``+?#`` ) when setting or modifying tag timestamps or tag properties.
```

Comment:
Tagprops use the `?=` syntax like props rather than `+?#` like tags

Reply:
Updated